src/build/__init__.py

```diff
@@ -35,6 +37,13 @@
 ]
 
 
+_WHEEL_NAME_REGEX = re.compile(
+    r'(?P<distribution>.+)-(?P<version>.+)'
+    r'(-(?P<build_tag>.+))?-(?P<python_tag>.+)'
+    r'-(?P<abi_tag>.+)-(?P<platform_tag>.+).whl'
```

Comment:
Escape the dot: `\.whl`.

Can we use `parse_wheel_filename` from `packaging.utils`?  Also missing coverage on line 358 'cause of this.

Reply:
Because it is fairly simple and allows us to limit the runtime dependency.